src/main/kotlin/three/consulting/epoc/controller/EmployeeController.kt

```diff
@@ -42,4 +42,8 @@ class FirebaseEmployeeController(
     @PreAuthorize("hasAuthority('ADMIN')")
     @PutMapping(consumes = [APPLICATION_JSON_VALUE], produces = [APPLICATION_JSON_VALUE])
     fun updateEmployeeForId(@Valid @RequestBody employee: EmployeeDTO) = firebaseService.updateEmployeeAndFirebaseRole(employee)
+
+    @PreAuthorize("hasAuthority('ADMIN')")
+    @GetMapping(value = ["/sync-firebase-users"], consumes = [ALL_VALUE])
+    fun syncFirebaseUsers() = firebaseService.syncAllFirebaseUsers()
```

Comment:
Should you also return the newly synced users from the database? That would mean one less request from the frontend. Or if you rather keep these two actions separate, this would be more of a `HEAD` than `GET` request as you're not retrieving a body of data, though there's no real specification in http which method to use as a trigger event

Reply:
I think it does what it supposed to do - sync users between db and firebase. If it success it will return 200, right? Otherwise it will throw an error. Would it be good idea to make endpoint return either 1) no changes (-> do nothing) or 2) changes (-> request synced employees in frontend)

Reply:
I guess it's fine as is. But there's some unnecessary mixing between terms we use, both employee and user are used in code. This could just be `/employee-sync` as people in firebase and app db represent the same thing.

Reply:
After a good night's sleep I made some changes. It was trivial to check if employee/user have any changes so returning a list of updated employees could work. Should we use "employee" term always in the code instead of "user"? I suppose it is possible to have users that are not employees at some point.

Reply:
Yeah but we're building the app for our internal use right now I vote for keeping things as simple as possible where we can